js/tidex.js

```diff
@@ -528,6 +528,10 @@ module.exports = class tidex extends Exchange {
             'filled': filled,
             'fee': undefined,
             // 'trades': this.parseTrades (order['trades'], market),
+            'info': undefined,
```

Comment:
This is a breaking change considering the `extend` below. Need to switch the ordering in the `extend`...

Reply:
... everywhere...

Reply:
imo there shouldn't be an extend there, just a regular assignment

Reply:
@frosty00 yep, you're right, an assignment would be better